### ISSUE TITLE
[BUG] Add alt text to Cover images in other features

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5550]
**Describe the bug**
This is a bug split from Epic https://github.com/rokwire/illinois-app/issues/5547. After consulting with Vanessa, it made sense to split the remediation effort by the affected app areas.

The scope of this bug covers: 

    - My Courses detail panel default image*
    - Dining locations detail panel default images*
    - MyMcKinley detail panel default images

**To Reproduce**

1. Enable Screen reader
2. Go to a 'My Courses' you are a member of that has a cover photo
3. Gesture through ensuring that image is announcing alt text to user
(Repeat for Dining locations and MyMcKinley detail panel default images)

**Expected behavior**
All non decorative images need alternative text to convey meaning to user.

**Actual behavior**
The images added before the Alt text field was created all have NULL alt text.

**Screenshots**
I did not have Courses with images and could not access MyMcKinley for appointments with image either. Will need help finding the data that has NULL to test. 

![Image](https://github.com/user-attachments/assets/86c557f1-8dc2-4373-974b-bce1c9647118)

![Image](https://github.com/user-attachments/assets/645596c0-1302-4134-b16a-7b4b591b959d)

This alt text should work regardless of platform or device. Therefore not including info of specific test environments.

**Additional context**
The images in Dining were all skipped when navigating with screen reader - Was this by design? I will need to understand to test this issue.

**Comment by TodorBachvarov** [comment]
@acovher2, All Images with missing Alt text will be skipped and will be treated as decorative images.

**Comment by acovher2** [comment]
@TodorBachvarov when it announces "Sports image" was this put in by someone or something automatically done? I was not sure if perhaps the admin users were doing this and needed help text on a field or not.  I am asking because repeating this alt text in itself gives 0 information to the user and these may as well be marked decorative.  Please let me know.
